test/Feature/WaveOps/WaveGetLaneIndex.test

```diff
@@ -0,0 +1,44 @@
+#--- source.hlsl
+RWStructuredBuffer<uint> Out : register(u0);
+
+[numthreads(4, 1, 1)]
+void main(uint3 threadID : SV_DispatchThreadID) {
+  Out[threadID.x] = WaveGetLaneIndex();
+}
+
+//--- pipeline.yaml
+
+---
+Shaders:
+  - Stage: Compute
+    Entry: main
+    DispatchSize: [1, 1, 1]
```

Comment:
Would be good to try larger DispatchSizes here so that WaveGetLaneIndex() doesn't always equal threadID.x.